src/pages/Recipe/Recipe.tsx

```diff
@@ -0,0 +1,9 @@
+import { RecipeCard } from './components/RecipeCard';
+
+export const Recipe: React.FC = () => {
+  return (
+    <>
+      <RecipeCard />
+    </>
+  );
```

Comment:
Здесь один компонент лишний кажется
```suggestion
  return <RecipeCard />;
```

Reply:
done